.teamcity/settings/extensions.kt

```diff
@@ -17,6 +17,15 @@ fun ParametrizedWithType.javaHome(os: Os, javaVersion: JavaVersion) {
     param("env.JAVA_HOME", javaVersion.javaHome(os))
 }
 
+fun ParametrizedWithType.androidHome(os: Os) {
+    val androidHome = when (os) {
+        Os.linux, Os.macos -> "/opt/android/sdk"
+        Os.windows -> """C:\Program Files\android\sdk"""
+    }
+    param("env.ANDROID_HOME", androidHome)
```

Comment:
Let's only set `ANDROID_SDK_ROOT` and not `ANDROID_HOME`. `ANDROID_HOME` is only for very old AGP versions.

Reply:
done